crates/router/src/core/errors.rs

```diff
@@ -259,45 +259,44 @@ pub enum WebhooksFlowError {
     #[error("Webhook details for merchant not configured")]
     MerchantWebhookDetailsNotFound,
     #[error("Merchant does not have a webhook URL configured")]
-    MerchantWebhookURLNotConfigured,
-    #[error("Payments core flow failed")]
-    PaymentsCoreFailed,
```

Comment:
Are these unused, why are these removed?

Reply:
Yes, the variants were not used anywhere. We can always add new variants as and when required.